### ISSUE TITLE
Deploying to k8s: json: expect char '"' but got char '1'

### ISSUE THREAD

**Issue by koalalorenzo** [gluster/gluster-kubernetes#343]
Hello I am currently running into this problem:
```
kube-templates# kubectl apply -f ./glusterfs-daemonset.yaml
Error from server (BadRequest): error when creating "./glusterfs-daemonset.yaml": DaemonSet in version "v1beta1" cannot be handled as a DaemonSet: [pos 3092]: json: expect char '"' but got char '1'
```

I am using the latest version of k8s and kubectl (1.7.5)

**Comment by grebois** [comment]
+1

**Comment by grebois** [comment]
Fix -> https://github.com/gluster/gluster-kubernetes/pull/345

**Comment by koalalorenzo** [comment]
Awesome! :D

**Comment by koalalorenzo** [comment]
Closing the issue when the PR will be merged!

**Comment by YorikSar** [comment]
So this should be fixed by #346, right?

**Comment by jarrpa** [comment]
Ah, yes! Thank you. :) Closing this issue, it can be reopened if the problem somehow persists (which it shouldn't ;) ).
